src/MultiFormGoals/resources/views/progressbar.php

```diff
@@ -34,3 +38,9 @@
 		<?php endif; ?>
 	</div>
 </div>
+<script>
+	const container = document.getElementById('<?php echo $uniqid; ?>')
+	const content = container.innerHTML
+	let shadow = container.attachShadow({mode: 'closed'});
```

Comment:
I believe we want this to be open. Open doesn't mean it will suddenly be affect by parent styles, but it means the the shadow contents can be affected, which we want — they just have to be very intentionally extended.

Reply:
I remember now - updated.